HW20/roles/hw201/tasks/main.yaml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Add Docker CE repo ubuntu
+  become: true
+  block:
+    - name: Add Docker GPG apt Key
+      apt_key:
+        url: https://download.docker.com/linux/ubuntu/gpg
+        state: present
+    - name: Add docker repository
+      apt_repository:
+        repo: deb https://download.docker.com/linux/ubuntu focal stable
+        state: present
+    - name: update apt cache
+      apt: update_cache=yes
+  when: ansible_distribution == 'Ubuntu'
+
+- name: Add Docker CE repo centos
+  become: true
+  get_url:
```

Comment:
Use loop here to be able to download more than one file

Reply:
I'm not just downloading the file. I also save it to the repositories folder. It only works like that. I do not need to download anything else to the folder with repositories.  yum_repository corrupts the repository.

Reply:
It is not just downloading file. yum_repository corrupts docker repostory file. So i exclusively save docker repo file to repo folder. Maybe in other cases yum_repository will be work fine.